engine/src/flutter/shell/platform/darwin/ios/platform_view_ios.mm

```diff
@@ -207,6 +207,7 @@ new PlatformMessageHandlerIos(task_runners.GetPlatformTaskRunner())) {}
   }
   [owner_controller_.platformViewsController reset];
   [owner_controller_.restorationPlugin reset];
+  [owner_controller_.textInputPlugin reset];
```

Comment:
just curious - the `reset` method of text input plugin was defined but never called before?

Reply:
No `reset` is new, but it calls the existing `hideTextInput` method which is private and responds to the framework's [`TextInput.hide`](https://github.com/flutter/flutter/blob/c5864174d060cc619f1896e05c3284016aade2cd/packages/flutter/lib/src/services/text_input.dart#L2510) message.